ui/raidboss/data/04-sb/dungeon/bardams_mettle.js

```diff
@@ -0,0 +1,313 @@
+'use strict';
+
+// Bardam's Mettle
+
+[{
+  zoneRegex: {
+    en: /^Bardam's Mettle$/,
+    cn: /^巴儿达木霸道$/,
+    ko: /^바르담 패도$/,
+  },
+  zoneId: ZoneId.BardamsMettle,
+  timelineFile: 'bardams_mettle.txt',
+  timelineTriggers: [
+    {
+      id: 'Bardam\'s Mettle Feathercut',
+      regex: /Feathercut/,
+      beforeSeconds: 4,
+      condition: function(data) {
+        return data.role == 'tank' || data.role == 'healer';
+      },
+      response: Responses.tankBuster(),
+    },
+  ],
+  triggers: [
+    {
+      id: 'Bardam\'s Mettle Rush',
+      netRegex: NetRegexes.tether({ id: '0039' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Extend the tether',
+      },
+    },
+    {
+      id: 'Bardam\'s Mettle War Cry',
+      netRegex: NetRegexes.startsUsing({ id: '1EFA', source: 'Garula', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '1EFA', source: 'Garula', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '1EFA', source: 'Garula', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '1EFA', source: 'ガルラ', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '1EFA', source: '加鲁拉', capture: false }),
+      condition: function(data) {
+        return data.role == 'healer';
```

Comment:
Conditions.caresAboutAOE or a comment explaining why this?

Reply:
I'm sure I had a reason somewhere, but I can't remember what it was, so `Conditions` it will be.